konrad/core.py

```diff
@@ -89,8 +88,8 @@ def __init__(self, atmosphere, timestep='3h', max_duration='5000d',
         self.ozone = utils.return_if_type(ozone, 'ozone',
                                           Ozone, OzonePressure())
 
-        self.humidity = utils.return_if_type(humidity, 'humidity',
-                                             Humidity, FixedRH())
+        if humidity is None:
+            self.humidity = FixedRH()
```

Comment:
You need
`else:
     self.humidity = humidity`
after this, in case we want to use something other than the default.

Reply:
Yes, of course!